subsys/usb/host/class/usbh_hub.c

```diff
@@ -0,0 +1,186 @@
+/*
+ * Copyright 2025 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/kernel.h>
+#include <zephyr/logging/log.h>
+#include <zephyr/usb/usbh.h>
+#include <zephyr/sys/byteorder.h>
+#include "usbh_device.h"
+#include "usbh_ch9.h"
+#include "usbh_hub.h"
+
+LOG_MODULE_REGISTER(usbh_hub, CONFIG_USBH_LOG_LEVEL);
+
+/**
+ * @brief Common hub control request function
+ */
+static int usbh_hub_class_request_common(struct usbh_hub_instance *hub_instance,
+					 uint8_t request_type, uint8_t request,
+					 uint16_t wvalue, uint16_t windex,
```

Comment:
Exceptionally, for variable names that come from the USB standard, you can use CamelCase:
```suggestion
					 uint16_t wValue, uint16_t wIndex,
```